dashboard/config/initializers/preload_schools.rb

```diff
@@ -0,0 +1,11 @@
+require 'cdo/school_autocomplete'
+
+# Preload the SchoolAutocomplete singleton instance.
+# Skip if this is running a Rake task (e.g. rake db:setup) or when caching is disabled
+
+if File.basename($0) != 'rake' && !Rails.application.config.skip_preload_schools
+  startTime = Time.now
+  SchoolAutocomplete.instance
+  endTime = Time.now
+  puts "[#{endTime.strftime('%c')}] Preloaded schools in #{endTime - startTime} second(s)"
```

Comment:
Pls use `CDO.log.info` rather than `puts`

Reply:
Updated. I wasn't clear before that the logs were going to `dashboard/log`. Thanks!